e2e-tests/wait-for-start.sh

```diff
@@ -1,7 +1,37 @@
 #!/usr/bin/env bash
 
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+######## WAIT FOR START #############
+# Script used in e2e-test that waits 
```

Comment:
nit: Any reason for keeping these comment lines too short? I usually break at around 80 (or 100 depending on the context) to keep the vertical length of the file shorter.

Reply:
Fixed